packages/migrate-config/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@eslint/migrate-config",
+  "version": "0.0.0",
+  "description": "Configuration migration for ESLint",
+  "type": "module",
+  "bin": {
+    "eslint-migrate-config": "./src/migrate-config-cli.js"
+  },
+  "files": [
+    "src"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
its public by default?

Reply:
Not for scoped packages:

https://docs.npmjs.com/creating-and-publishing-scoped-public-packages#publishing-scoped-public-packages